activesupport/lib/active_support/core_ext/array/pluck.rb

```diff
@@ -0,0 +1,20 @@
+class Array
+  # Extracts values from array of hashes.
+  #
+  #   x = [{ a: 1, b: 2 }, { a: 10, b: 20 }]
+  #
+  #   instead of
+  #   x.map { |h| h[:a] }
+  #
+  #   x.pluck(:a) => [1, 10]
+  #   x.pluck(:a, :b) => [[1, 2], [10, 20]]
+  def pluck(*keys)
+    if keys.size > 1
```

Comment:
`map` with `slice` could look more elegant and compact, are there any problems with using it?